backend/api.py

```diff
@@ -0,0 +1,37 @@
+import flask
+from flask import request, jsonify
+from query import query
+from add import add
+import ast
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+@app.route('/', methods=['GET'])
+def home():
+    return '''<h1>DOTS</h1>
+<p></p>'''
+
+
+@app.errorhandler(404)
+def page_not_found(e):
+    return "<h1>404</h1><p> 404 :/ </p>", 404
+
+
+
+@app.route('/dots/organisations/query', methods=['POST'])
+def query_organisations():
+    body = ast.literal_eval(request.data.decode('utf-8'))
+    res = query(body['filters'])
+    return jsonify(res), 201
+
+
+
+@app.route('/dots/organisations/add', methods=['POST'])
```

Comment:
I suggest use blueprint to build a more modular api...
https://dev.to/dendihandian/modular-flask-app-with-blueprints-186